_data/education.yml

```diff
@@ -5,6 +5,12 @@ websites:
       img: academicearth.png
       tfa: No
 
+    - name: Antagning.se
+      url: https://www.antagning.se
+      img: antagning.png
+      twitter: antagningse
+      tfa: no
```

Comment:
As with #1724, would you be able to change tfa: no to tfa: No by any chance? This ensures that each line is consistent across every entry.